packages/nuxt/src/vite/addServerConfig.ts

```diff
@@ -29,10 +31,11 @@ export function addServerConfigToBuild(
      * When the build process is finished, copy the `sentry.server.config` file to the `.output` directory.
      * This is necessary because we need to reference this file path in the node --import option.
      */
-    nuxt.hook('close', async () => {
-      const rootDirResolver = createResolver(nuxt.options.rootDir);
+    nitro.hooks.hook('close', async () => {
+      const rootDirResolver = createResolver(nitro.options.rootDir);
+      const serverDirResolver = createResolver(nitro.options.output.serverDir);
       const source = rootDirResolver.resolve('.nuxt/dist/server/sentry.server.config.mjs');
```

Comment:
In SolidStart, I can get the build directory via `nitro.options.buildDir`, does Nuxt's nitro config not have that exposed?